filecoin-proofs/src/api.rs

```diff
@@ -481,3 +589,121 @@ fn pad_safe_fr(unpadded: &FrSafe) -> Fr32Ary {
     res[0..31].copy_from_slice(unpadded);
     res
 }
+
+#[test]
```

Comment:
TODO: Put this test [into a `tests` module](https://doc.rust-lang.org/book/ch11-03-test-organization.html).

```
#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test_generate_piece_commitment() {
```